src/index.js

```diff
@@ -164,11 +164,14 @@ const Index = () => {
                             <Route path="delete" component={userIsAuthenticated(TeamsController)} />
                         </Route>
                     </Route>
-                    <Route path={`${rootPath}/interactives`}>
-                        <IndexRoute component={userIsAuthenticated(InteractivesController)} />
-                        <Route path="create" component={userIsAuthenticated(InteractivesFormController)} />
-                        <Route path="edit/:interactiveId" component={userIsAuthenticated(InteractivesFormController)} />
-                    </Route>
+                    {config.enableNewInteractives === true && (
+                        <Route path={`${rootPath}/interactives`}>
```

Comment:
would be great if you can aviod having nested rotes as this only will need to be refacored once we will be able to update the react-route